resolwe/test_helpers/test_runner.py

```diff
@@ -0,0 +1,205 @@
+""".. Ignore pydocstyle D400.
+
+===================
+Resolwe Test Runner
+===================
+
+"""
+import os
+import re
+import subprocess
+
+import yaml
+
+from django.core.management.base import CommandError
+from django.test.runner import DiscoverRunner
+
+from resolwe.flow.finders import get_finders
+from resolwe.test.utils import generate_process_tag
+
+SPAWN_PROCESS_REGEX = re.compile(r'run\s+\{.*?["\']process["\']\s*:\s*["\'](.+?)["\'].*?\}')
+
+
+class ResolweRunner(DiscoverRunner):
+    """Resolwe test runner."""
+
+    def __init__(self, *args, **kwargs):
+        """Initialize test runner."""
+        self.only_changes_to = kwargs.pop('only_changes_to', None)
+        self.changes_file_types = kwargs.pop('changes_file_types', None)
+
+        super(ResolweRunner, self).__init__(*args, **kwargs)
+
+    @classmethod
+    def add_arguments(cls, parser):
+        """Add command-line arguments."""
+        super(ResolweRunner, cls).add_arguments(parser)
+
+        parser.add_argument(
+            '--only-changes-to', dest='only_changes_to',
+            help="Only test changes against given Git commit reference"
+        )
+
+        parser.add_argument(
+            '--changes-file-types', dest='changes_file_types',
+            help="File which describes what kind of changes are available"
+        )
+
+    def run_tests(self, test_labels, **kwargs):
+        """Run tests."""
```

Comment:
Please, also document all parameters with Sphinx's reST fields notation.

Applies to all public functions.